scripts/apps/archive/directives/HtmlPreview.ts

```diff
@@ -52,9 +52,11 @@ export function HtmlPreview($sce, $timeout) {
         templateUrl: 'scripts/apps/archive/views/html-preview.html',
         link: function(scope, elem, attrs) {
             scope.$watch('sdHtmlPreview', (html) => {
-                scope.html = stringIsHtml(html)
-                    ? $sce.trustAsHtml(html)
-                    : $sce.trustAsHtml(plainTextToHtml(html));
+                if (html && typeof html === 'string') {
```

Comment:
You don't need `html &&` part. You only need to check if it's not null if you accessing object properties.